<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageFilter_Gui</class>
 <widget class="QWidget" name="ImageFilter_Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Gaussian Filter</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="Kernelsize_label">
         <property name="text">
          <string>Kernel size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="SigmaX_label">
         <property name="text">
          <string>SigmaX</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="Kernelsize_comboBox">
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="SigmaX_comboBox">
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="GaussisanFilter_pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Medianfilter_groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Median filter</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QComboBox" name="MedianKernelsize_comboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>74</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="MedianKernelsize_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>74</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kernel size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MedianFilter_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="LaplacianFilter_groupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Laplacian Filter</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="LaplacianFilter_pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
